api/cookie.go

```diff
@@ -36,8 +36,8 @@ var (
 // writeCookie is a helper function that writes the given JWT token as a
 // secure cookie.
 func writeCookie(w http.ResponseWriter, token string, exp int64) error {
-	if exp <= 0 || time.Unix(exp, 0).Before(time.Now()) {
-		exp = time.Now().Unix()
+	if exp <= 0 || time.Unix(exp, 0).Before(time.Now().UTC()) {
```

Comment:
why is exp an `int64` and not a `uint64`?

Reply:
Go's `time.Now()` returns int64 and `time.Unix` takes int64. I didn't see a reason to convert twice.

Reply:
Hm, I guess the use of `int64` is there to allow referenced to times before Jan 1st, 1970.
Running `fmt.Println(time.Unix(-1000000, 0))` prints out `1969-12-20 11:13:20 +0100 CET`